trade-finance-manager-api/src/graphql/reducers/mappings/facilities/mapFacilityValueExportCurrency.js

```diff
@@ -0,0 +1,27 @@
+const api = require('../../../../v1/api');
+const { formattedNumber } = require('../../../../utils/number');
+const { amendmentChangeValueExportCurrency, latestAmendmentValueAccepted } = require('../../../helpers/amendment.helpers');
+
+// maps facility value (in original currency) and checks if amendment change on that value
+const mapFacilityValueExportCurrency = async (facility) => {
+  if (facility) {
+    const { _id, facilitySnapshot } = facility;
+
+    const { currency, value } = facilitySnapshot;
+
+    // checks if completed amendment which is accepted
+    const latestCompletedAmendment = await api.getLatestCompletedAmendment(_id);
```

Comment:
same as the other file - this can be a `new Promise()`. that way you don't have to introduce the async/await in the other files

Reply:
done

Reply:
decided to use await instead